src/lib/eet/meson.build

```diff
@@ -1,4 +1,9 @@
 eet_deps = [crypto, jpeg, rg_etc]
+if not sys_windows
+  eet_deps += m
+  assert(eet_deps[-1] == m)
```

Comment:
Is this assert really needed?

Reply:
it was more like a to see if I was working correctly with the meson Array, I can remove all the new ones if you think it is better